en/docs/reference/connectors/creatio-connector/creatio-connector-reference.md

```diff
@@ -0,0 +1,623 @@
+
+    If the given ID does not exist in the CreatioCRM, this will return a 404 HTTP status code.
+
+    |Field|Type| |Description|Remarks|
+    |:----|:----|:----|:----|:----|
+    |BPMCSRF|String| |The CSRF Token from the Auth response|Ex: bHZfVOIO4.iSuXwMebGibes|
+    |Set-cookie|String| |Value of Username cookie from the Auth response|Ex: UserName=83|117|112|101|114|118|105|115|111|114; expires=Sat, 07-May-2022 08:46:12 GMT; path=/; secure; HttpOnly|
+    |Cookie|String| |Cookie Value from the Auth response| |
+
+    **Sample configuration**
+
+    ```xml
+    <creatiocrm.init>
+        <hostName>https://creatio-hostname</hostName>
+        <password>creatio-username</password>
+        <username>creatio-password</username>
+    </creatiocrm.init>
+    <creatiocrm.accountGet>
+        <id>{$ctx:uri.var.id}</id>
+    </creatio.accountGet>
+
+    ```
+
+??? note "Update Account"
+    The accountUpdate operation is used to update the existing account details in the CreatioCRM. 
+
+    If the given ID does not exist in the CreatioCRM this will create a new account with the given ID.
+
+    |Field|Type| |Description|Remarks|
+    |:----|:----|:----|:----|:----|
+    |BPMCSRF|String| |The CSRF Token from the Auth response|E.g., bHZfVOIO4.iSuXwMebGibes|
+    |Set-cookie|String| |Value of Username cookie from the Auth response|E.g., UserName=83|117|112|101|114|118|105|115|111|114; expires=Sat, 07-May-2022 08:46:12 GMT; path=/; secure; HttpOnly|
+    |Cookie|String| |Cookie value from the Auth response| |
+
+    **Sample configuration**
+
+    ```xml
+    <creatiocrm.init>
+        <hostName>https://creatio-hostname</hostName>
+        <password>creatio-username</password>
+        <username>creatio-password</username>
+    </creatiocrm.init>
+    <creatiocrm.accountUpdate/>
+        <id>{$ctx:uri.var.id}</id>
+    </creatio.accountUpdate>
+
+    ```
+
+    **Sample request**
+
+    ```json
+
+    {
+        "Name": "API 1",
+        "ProcessListeners": 0,
+        "Phone": "+1 206 480 3801",
+        "AdditionalPhone": "+1 206 480 4495",
+        "Fax": "",
+        "Web": "www.infocom-global.com",
+        "Address": "48 Pilgrim Street",
+        "Notes": ""
+    }
+
+    ```
+
+??? note "Delete Account"
+    The accountDelete operation is used to delete an account that already exists in the CreatioCRM with a given unique ID.
+    
+    If the given ID does not exist in the CreatioCRM, this will return a 404 HTTP status code.
+
+    |Field|Type| |Description|Remarks|
+    |:----|:----|:----|:----|:----|
+    |BPMCSRF|String| |The CSRF Token from the Auth response|E.g., bHZfVOIO4.iSuXwMebGibes|
+    |Set-cookie|String| |Value of Username cookie from the Auth response|E.g., UserName=83|117|112|101|114|118|105|115|111|114; expires=Sat, 07-May-2022 08:46:12 GMT; path=/; secure; HttpOnly|
+    |Cookie|String| |Cookie value from the Auth response| |
```

Comment:
The cookies will be added by the connector hence the end user doesn't need to configure them. Therefore shall we remove these `Request Header attributes` and have only `Request Parameters` as defined in the internal doc? Please check other operations as well

Reply:
As per the offline discussion `Request Header attributes` should be optional. Therefore shall we mention them as optional instead of removing them? @samgnaniah